manifests/satellite6.pp

```diff
@@ -0,0 +1,19 @@
+# Class: satellite6
+#   Install the SAtellite 6 flavor of the robottelo_slave
+#
+class robottelo_slave::satellite6 {
```

Comment:
This is intheory for downstream, but there is nothing that says you couldn't apply and run this upstream if you wanted a slave capable of running the robottelo test suites. Which is why I prefer something that describes that this class does instead of what it's intended for. In other words, to me this particular class is about configuring a slave to handle building not necessarily to configure a slave to be a "satellite6" slave.